src/main/webapp/map.js

```diff
@@ -49,21 +39,60 @@ function createMap() {
   }
 }
 
+// Request nearby coffee shop info
+// Unit of radius: metres. Maximum allowed is 50000 metres
```

Comment:
Suggestion: Add a comment indicating whether the max 5km is a Geocode/Maps API or a TYD Coffee restriction.